public/components/create_report/delivery/delivery.tsx

```diff
@@ -13,60 +13,202 @@
-          <DeliveryChannelRadio/>
-        </EuiFormRow>
-        <EuiFormRow label="Recipients">
-          <DeliveryRecipientsBox/>
+export function ReportDelivery() {
+  const [emailCheckbox, setEmailCheckbox] = useState(false);
+  const [emailRecipients, setEmailRecipients] = useState();
+  const [emailSubject, setEmailSubject] = useState('');
+  const [emailBody, setEmailBody] = useState('');
+  const [includeReportAsAttachment, setIncludeReportAsAttachment] = useState(
+    false
+  );
+  const [insertPlaceholder, setInsertPlaceholder] = useState(false);
+
+  const onChangeEmailCheckbox = (e) => {
+    setEmailCheckbox(e.target.checked);
+  };
+
+  const onChangeEmailRecipients = (e) => {
+    setEmailRecipients(e);
+  };
+
+  const onChangeEmailSubject = (e) => {
+    setEmailSubject(e.target.value);
+  };
+
+  const onChangeEmailBody = (e) => {
+    setEmailBody(e.target.value);
+  };
+
+  const onChangeIncludeReportAsAttachment = (e) => {
+    setIncludeReportAsAttachment(e.target.checked);
+  };
+
+  const onInsertPlaceholderClick = () =>
+    setInsertPlaceholder((insertPlaceholder) => !insertPlaceholder);
+  const closeInsertPlaceholder = () => setInsertPlaceholder(false);
+
+  const onCreateEmailRecipient = (searchValue, flattenedOptions = []) => {
+    const normalizedSearchValue = searchValue.trim().toLowerCase();
+
+    if (!normalizedSearchValue) {
+      return;
+    }
+
+    const newOption = {
+      label: searchValue,
+    };
+
+    // Create the option if it doesn't exist.
+    if (
+      flattenedOptions.findIndex(
+        (option) => option.label.trim().toLowerCase() === normalizedSearchValue
+      ) === -1
+    ) {
+      email_recipient_options.push(newOption);
+    }
+
+    // Select the option.
+    setEmailRecipients([...emailRecipients, newOption]);
+  };
+
+  const placeholderInsert = (
+    <EuiText size="xs">
+      <EuiLink onClick={onInsertPlaceholderClick}>Insert placeholder</EuiLink>
+    </EuiText>
+  );
+
+  const InsertPlaceholderPopover = () => {
+    return (
+      <div>
+        <EuiPopover
+          button={placeholderInsert}
+          isOpen={insertPlaceholder}
+          closePopover={closeInsertPlaceholder}
+        >
+          <EuiListGroup listItems={insert_placeholder_options} />
+        </EuiPopover>
+      </div>
+    );
+  };
+
+  const EmailDelivery = () => {
+    return (
+      <div>
+        <EuiFormRow
+          label="Email recipients"
+          helpText="Select or add recipients"
+        >
+          <EuiComboBox
+            placeholder={'add users here'}
+            options={email_recipient_options}
+            selectedOptions={emailRecipients}
+            onChange={onChangeEmailRecipients}
+            onCreateOption={onCreateEmailRecipient}
+          />
         </EuiFormRow>
-        <EuiFormRow label="Email Subject">
+        <EuiSpacer />
+        <EuiFormRow label="Email subject">
```

Comment:
is adding `Email` prefix in title mandatory ?

Reply:
`Email` prefix is in the UX mockups